CodeOfConduct.md

```diff
@@ -16,11 +16,46 @@ WebAssembly operates under the W3C's
 > vehicle for better identity of the organization.
 
 We hope that our community group act according to these guidelines, and that
-participants hold each other to these high standards. If you have any questions
-or are worried that the code isn't being followed, please contact the Community
-Group's chairs at `team-wasm-chairs@w3.org` (note: this list is also
-visible to W3C staff). For very serious concerns, the W3C has [procedures][]
-allowing you to access its ombuds directly and confidentially.
+participants hold each other to these high standards. 
+
+If you are worried that the code isn't being followed, or are witness to a violation but are
+uncomfortable speaking up,please contact the Community Group's chairs at
+`team-wasm-chairs@w3.org` (note: this list is also visible to W3C staff). 
+
+In cases where there is repeated disregard for the CEPC, it is possible to incur disciplinary
+action. The points below outline a series of steps that will be taken in case of violations
+to ensure a positive work environment in the WebAssembly CG. 
+
+ - **Informal warning**   
+   An informal or verbal warning can be issued in any relevant medium of communication callin 
+   out inappropriate behavior as a violation of the CEPC
+   (examples: responses to github issues or PRs, discord, email, meetings).
+
+ - **Corrective action and/or counseling**   
+   Informal meetings, or email discussions where problematic behavior is addressed,
+   and suggestions for constructive ways forward are offered. A mediation process can also be an
+   option if relevant to the situation and both parties agree. 
```

Comment:
Who is the second party here? The original reporter or possibly someone else?

Reply:
It depends on the the situation, it could be the original reporter, sometimes the reporter could be a different person than the one targeted, it could also be a relevant group of people.